### PR TITLE
Add windows support

### PR DESCRIPTION
**Pull request by TigerWolf**
Python adds a \ in windows when checking for the path and starts with 'c' instead of 'file'

https://stackoverflow.com/questions/43911052/urlparse-on-a-windows-file-scheme-uri-leaves-extra-slash-at-start

I dont normally program python so this might not be the right way to do this but thought I would share this fix to make it work on windows (I know)

### CODE REVIEW

**Comment by mueslo** [comment]
Hey, thanks for the contribution. 

> Python adds a \ in windows when checking for the path and starts with 'c' instead of 'file'

Which means Python is working correctly as per any sane URI specification. The problem here is windows's shitty tradition of drive names.

So there are some issues:

1) Please do not add any new dependencies unless required. Use the future.moves.urllib library indirection currently in use.
2) You are only accounting for the C drive, while drive names could be any single-letter name. 
3) It adds complexity/redundancy. Except for this issue does it even work on windows? If so, it's probably better to change it to check for http(s)/file URI and in any other case assume it is a OS-usable path and open it directly.

File: pythonbits/imgur.py [hunk 1]
```diff
@@ -99,7 +99,8 @@ def fetch_access_token(self, grant_type, value):
     def get_auth_headers(self):
         return {"Authorization": "Bearer %s" % self.access_token}
 
-
+import urllib
+import urllib.request
```
Comment (mueslo) [inline pythonbits/imgur.py:103]: remove

File: pythonbits/imgur.py [hunk 2]
```diff
@@ -115,6 +116,8 @@ def upload(self, image):
             params['data'] = {'image': image}
         elif urlparse(image).scheme in ('file', ''):
             params['files'] = {'image': open(urlparse(image).path, "rb")}
+        elif urlparse(image).scheme in ('c'):
```
Comment (mueslo) [inline pythonbits/imgur.py:119]: generalize

**Comment by TigerWolf** [comment]
I have abandoned this as I am now just using the linux subsystem for windows instead :D

**Pull request closed without merging**